packages/gatsby-plugin-manifest/README.md

```diff
@@ -475,3 +475,30 @@ npm install gatsby-plugin-sharp gatsby-plugin-manifest gatsby-remark-images-cont
 ```
 
 If updating these doesn't fix the issue, your project probably uses other plugins from the community that depend on a different version of `sharp`. Try running `npm list sharp` or `yarn why sharp` to see all packages in the current project that use `sharp` and try updating them as well.
+
+### APACHE: Error while trying to use the following icon from the Manifest
```

Comment:
What setup are you using for this? I tried checking the file references in an Ubuntu 18.04 install and I don't have `httpd`. The file references don't exist, and the command to restart Apache is `sudo systemctl restart apache2`.

Reply:
I apologise for the error: I'm usually on some version of Ubuntu as well, but for this I was on a server running a red hat installation. Should I add directions for Debian systems as well as RH? Or should we just delete the directions, but keep the tip about modifying apache?

Reply:
I'll debian directions and specify the others as centOS / red hat in the next 24 hours. 

On Debian based systems:
Backup the /etc/apache2/mods-available/alias.conf file:
cp /etc/apache2/mods-available/alias.conf /etc/apache2/mods-available/alias.conf.back

Comment out the Alias /icons/ "/usr/share/apache2/icons/" row in /etc/apache2/mods-available/alias.conf file:
cat /etc/apache2/mods-available/alias.conf | grep "Alias /icons/"

Reload Apache service:
service apache2 reload